src/prototype_room_creepbuilder.js

```diff
@@ -122,7 +97,7 @@ Room.prototype.spawnCheckForCreate = function(creepsConfig) {
     var creep = this.memory.queue[0];
```

Comment:
didn't touch that yet... prefer to wait tooangel review but i'll maybe start to parse that part. dunno.